colossalai/fx/tracer/meta_patch/__init__.py

```diff
@@ -1,3 +1,5 @@
-from .registry import *
+from .patched_bias_addition_function import *
+from .patched_bias_addition_module import *
```

Comment:
Meanwhile, just wonder is it suitable to put it together with the patched function and module. The reason is that `patched_function` and `patched_module` are implemented to help the forward to execute the forward pass. Meanwhile, `patched_bias_addition_function` and `patched_bias_addition_module` are more like doing transform during graph tracing. I feel that there is a logical difference between them.

Reply:
I see, I will refactor the file structure.